src/app/api/manuscript.api.ts

```diff
@@ -95,9 +101,8 @@ export async function getManuscriptChanges(id: string): Promise<JSONObject[]> {
 // mock call
```

Comment:
This isn't a mock call anymore right? Might be worth cleaning up the comments...